os/Kconfig

```diff
@@ -104,6 +104,12 @@ config WINDOWS_MKLINK
 
 menu "Build Configuration"
 
+if ARCH_CHIP_RTL8730E
+config LIB_HOMEDIR
+	string "Home directory"
+	default "/"
+endif
```

Comment:
It looks much strange. Why is this necessary? Even this is necessary, I don't think here is a proper location for the config.